src/utils/format.ts

```diff
@@ -130,6 +130,13 @@ export const formatAddress = (address: string) => {
   return address.substring(0, 6) + '...' + address.substring(address.length - 4, address.length);
 };
 
+export const getENSname = (address: string) => {
+  //TO DO: need to set up a provider here.
```

Comment:
pass provider in as an arg. We'll always have access to it from actions

Reply:
What actions are you referencing, I just checked all store Actions and found nothing related to providers

Reply:
`wallet.provider`